packages/kit/src/core/utils.js

```diff
@@ -3,10 +3,13 @@ import path from 'path';
 import colors from 'kleur';
 import { copy } from '../utils/filesystem.js';
 import { fileURLToPath } from 'url';
+import { SVELTE_KIT } from './constants.js';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
+export const modules = path.resolve(`${SVELTE_KIT}/modules`);
```

Comment:
```suggestion
export const modules = path.posix.resolve(`${SVELTE_KIT}/modules`);
```

Reply:
I don't know why what I'm suggesting results in `import { start } from "/@fs/blah/kit/packages/kit/test/apps/basics/.svelte-kit/modules/client/start.js";` on Windows rather than `import { start } from "/.svelte-kit/dev/runtime/internal/start.js";` but I'm not especially anxious to dig into that right now.

In any case, the tests look like they're passing for me locally.